src/components/tabs/js/tabsDirective.js

```diff
@@ -131,27 +131,28 @@ function MdTabs ($$mdSvgRegistry) {
             '<md-icon md-svg-src="'+ $$mdSvgRegistry.mdTabsArrow +'"></md-icon> ' +
           '</md-next-button> ' +
           '<md-tabs-canvas ' +
-              'tabindex="{{ $mdTabsCtrl.hasFocus ? -1 : 0 }}" ' +
               'aria-activedescendant="{{$mdTabsCtrl.getFocusedTabId()}}" ' +
```

Comment:
This also needs to move down to the md-pagination-wrapper element, no? Or role="tablist" needs to stay on this one?